ansible_collections/arista/avd/molecule/evpn_underlay_rfc5549_overlay_ebgp/intended/configs/DC1-BL1B.cfg

```diff
@@ -251,6 +251,7 @@ router bgp 65105
       no neighbor EVPN-OVERLAY-PEERS activate
       neighbor UNDERLAY_PEERS activate
       neighbor UNDERLAY_PEERS next-hop address-family ipv6 originate
+      no neighbor 10.23.23.6 next-hop address-family ipv6
```

Comment:
@nathanmusser I think we need to keep this neighbor with the next-hop, since we have ipv6_enable. Now I understand why you asked about this a long time ago :-/